src/routes/wohnen/components/Offer.js

```diff
@@ -0,0 +1,137 @@
+// @flow
+
+import React from 'react'
+import WohnenOfferModel from 'modules/endpoint/models/WohnenOfferModel'
+import WohnenFormData from 'modules/endpoint/models/WohnenFormData'
+import styled from 'styled-components'
+import type { AccommodationType } from '../../../modules/endpoint/models/WohnenFormData'
+import type { TFunction } from 'react-i18next'
+import { translate } from 'react-i18next'
+import ListElement from '../../../modules/common/components/ListElement'
+
+type PropsType = {
+  offer: WohnenOfferModel<*>,
+  t: TFunction
+}
+
+const Header = styled.div`
+  padding: 15px 5px 5px;
+  font-weight: 700;
+`
+
+const RowTitle = styled.div`
+  flex: 50%;
+`
+
+const RowValue = styled.div`
+  flex: 50%;
+`
+
+const Row = styled.div`
+  display: flex;
```

Comment:
give them rows some private space :P 
sth like `margin: 10px 0;` to make them feel more comfortable...

Reply:
Other than that, looking good 👌